src/main/kotlin/javabot/operations/WeatherOperation.kt

```diff
@@ -16,7 +16,7 @@ class WeatherOperation @Inject constructor(bot: Javabot, adminDao: AdminDao, var
         val message = event.value
         if (message.toLowerCase().startsWith("weather ")) {
             val place = message.substring("weather ".length).trim()
-            val result = weatherDao.getWeatherFor(place)
+            val result = weatherDao.getWeatherFor(place, bot.javabotConfig)
```

Comment:
couldn't this config be injected in to the DAO itself?

Reply:
Why, yes, yes it can.